contextMenu.js

```diff
@@ -125,22 +126,21 @@
                         //update class name
                         className != undefined && elm.attr('class', className);
 
-                        //update image
+                        var imgIcon = elm.find('.iw-mIcon');
+                        if(imgIcon.length) imgIcon[0].remove();
```

Comment:
This should be imgIcon.remove(); 
imgIcon[0] will give DOM object and there is no remove method of DOM object (Thats a jquery method)